# Message corresponding to coordinates of the point after perspective projection
float64 x      # Coordinate of the point in the image plane in meter along X axis
float64 y      # Coordinate of the point in the image plane in meter along Y axis

